### DISCUSSION TITLE
Texture intensity from ND toward ED

### DISCUSSION THREAD

**Discussion by lyg1232** [mtex-toolbox/mtex#1387] [Ask Anything] [answered]
Hello!
I got some pole figures from mtex like in figure 5, but I don't know how to get the texture from ND to ED oblique angle, like in figure 6. So I want to ask for your help.
Thanks!!
![1](https://user-images.githubusercontent.com/81472625/159494333-297a55e5-1ca2-4684-b953-2eb90ae294ae.png)

**Comment by kilir** [comment, marked as answer]
Hi,
how about this:
```
% some pf from odf - different in your case
cs = crystalSymmetry('321');
odf = unimodalODF(orientation.rand(2,cs));
pdf = calcPDF(odf,Miller(1,0-1,1,cs));

% evaluate along line
rho = [0:3:90]*degree;
v = vector3d.byPolar(rho,0) % define to match your needs
plot(pdf)
hold on
plot(v,'MarkerFaceAlpha',0)
hold off

nextAxis
plot(rho/degree,eval(pdf,v)) % this is what you are looking for

```
Cheers,
Rüdiger
